service/http/service.go

```diff
@@ -36,11 +37,13 @@ func newServer(address string, router *mux.Router) *Server {
 		},
 		mux:                router,
 		topicSubscriptions: make([]*common.Subscription, 0),
+		authToken:          os.Getenv(common.AppAPITokenEnvVar),
```

Comment:
Store token in env?

Reply:
Yes, Need to provide other way, such as function?